src/components/YextEntityFieldSelector.tsx

```diff
@@ -87,8 +101,37 @@ export const YextEntityFieldSelector = <T extends Record<string, any>>(
               />
             </FieldLabel>
           )}
+          {shouldDisplayConstantValueField(props.filter.types) && (
+            <ToggleMode
+              entityFieldMode={entityFieldMode}
+              toggleEntityFieldMode={toggleEntityFieldMode}
+            />
+          )}
         </>
       );
     },
   };
 };
+
+const ToggleMode = ({
+  entityFieldMode,
+  toggleEntityFieldMode,
+}: {
+  entityFieldMode: boolean;
+  toggleEntityFieldMode: () => void;
+}) => {
+  return (
+    <div className="ve-mt-2 ve-w-full">
+      <Button onClick={toggleEntityFieldMode} variant="secondary">
+        {entityFieldMode ? (
+          <>
+            <Lock className="sm-icon" />
```

Comment:
Idk if we need a lock icon. I'd ask in the channel with a screenshot of what it looks like. It should be full width. It most likely should only change the text.

Reply:
Did we get an answer on this? Also, is the button 100% width now?

Reply:
Yup 100% width, and confirmed we'll remove the lock icon